ggml-sycl/common.hpp

```diff
@@ -261,16 +459,15 @@ struct ggml_backend_sycl_context {
 
     queue_ptr qptrs[GGML_SYCL_MAX_DEVICES][GGML_SYCL_MAX_STREAMS] = { { nullptr } };
 
-    explicit ggml_backend_sycl_context(int device) :
+    explicit ggml_backend_sycl_context(struct ggml_sycl_device_info &sycl_device_info, int device) :
         device(device),
         name(GGML_SYCL_NAME + std::to_string(device)) {
+            qptrs[device][0] = sycl_device_info.local_sycl_device_mgr->queues[device];
```

Comment:
breaks support for multi queues in single device

Reply:
Yes, update